server_admin/topics/users/user-registration.adoc

```diff
@@ -1,15 +1,16 @@
-[id="con-user-registration_{context}"]
+[id="assembly-user-registration_{context}"]
+= User Registration
```

Comment:
I think this works better as a single file with single procedure, not an assembly.

Reply:
I think mod guidelines would qualify this as an assembly. I think there 2 tasks that are identified for user registration. 
* You can enable user registration 
* You can complete a user registration form 

Is that correct? If so, I think this should be assembly

Reply:
Yes, now I get it.  Thanks for the correction. That makes sense.